tests/system/small/geopandas/test_geoseries.py

```diff
@@ -61,3 +66,24 @@ def test_geo_y(urban_areas_dfs):
         pd_result.astype(pd.Float64Dtype()),
         bf_result,
     )
+
+
+def test_geo_area_not_supported():
+    s = bigframes.pandas.Series(
+        [
+            Polygon([(0, 0), (1, 1), (0, 1)]),
+            Polygon([(10, 0), (10, 5), (0, 0)]),
+            Polygon([(0, 0), (2, 2), (2, 0)]),
+            LineString([(0, 0), (1, 1), (0, 1)]),
+            Point(0, 1),
+        ],
+        dtype=GeometryDtype(),
+    )
+    bf_series: bigframes.geopandas.GeoSeries = s.geo
+    with pytest.raises(
+        NotImplementedError,
+        match=re.escape(
+            f"GeoSeries.area is not supported. Use bigframes.bigquery.st_area(series), instead. {constants.FEEDBACK_LINK}"
+        ),
+    ):
+        bf_series.area()
```

Comment:
> mypy error: "Series" not callable  [operator]

Note: since `.area` is a "property", we don't call it as a function. We pretend it's just a normal attribute.

```suggestion
        bf_series.area
```

A bit more about properties: https://www.pythonmorsels.com/making-read-only-attribute/ and https://realpython.com/python-property/

It's very similar to what Java calls "getter" and "setter" functions, but in this case Python allows it to look like a normal attribute.